### ISSUE TITLE
[Offline] [Samsung] Not possible to work in offline mode as no connection error alert can't be closed

### ISSUE THREAD

**Issue by Dasha1990** [letsdoitworld/World-Cleanup-Day#223]
Device: Samsung S4 note 6.0.1 , SGS6 edge 
can not reproduce on Nexus 6p 8.0.1
Release build

Preconditions:
App was created and builded from master branch

Steps to reproduce:

Login to the app
Turn connection off
Check the app

Result:
no connection error is shown and it can't be closed
http://take.ms/LblTHU

can't reproduce on iOs

**Comment by Dasha1990** [comment]
Sorry for misunderstanding. Issue is that error is shown above all screens and user is not able to work inside the app in offline

**Comment by lynxlynxlynx** [comment]
So on ios the error isn't displayed at all or it is and is closable?

**Comment by Dasha1990** [comment]
you are right, there is a button which allows to close alert http://take.ms/ngeYv

**Comment by lynxlynxlynx** [comment]
Yeah, that's a bigger screen. I've reverted the style change, since this is much more important than cut-off translations. An improved fix can be found later.

But I guess this only masks the issue. You're not supposed to get the alert in offline mode, right?

**Comment by Dasha1990** [comment]
I should get error alert in offline mode, but I should be able to cancel or close in somehow

**Comment by aleksandr-wemakesoftware** [comment]
Does this happen only on a smaller devices?

In this case the bug is not related to offline mode, but instead to the misplaced buttons on a smaller screens.

**Comment by lynxlynxlynx** [comment]
This should be gone with the next rebuild then.

**Comment by Dasha1990** [comment]
This is happening on Samsung S4 note 1440x2560 pixels

**Comment by lynxlynxlynx** [comment]
@darjalsk this one should be gone too.
